resources/qml/WelcomePages/AddLocalPrinterScrollView.qml

```diff
@@ -78,98 +81,204 @@ Item
         id: localPrinterSelectionItem
         anchors.fill: parent
 
-        //Selecting a local printer to add from this list.
-        ListView
+        Column
         {
-            id: machineList
+            id: root
             width: Math.floor(parent.width * 0.48)
             height: parent.height
+            Item
+            {
+                width: root.width
+                height: filter.height
+                Cura.TextField
+                {
+                    id: filter
+                    width:parent.width
+                    implicitHeight: parent.height
+                    background: Rectangle {
+                        id: background
+                        color: UM.Theme.getColor("main_background")
+                        radius: 2
```

Comment:
Shouldn't this also be from a theme?

Reply:
a3911bdbfa47b0825d9b4980584fa9cf37ed52a1